ios/nskeyedarchiver/objectivec_classes.go

```diff
@@ -108,6 +108,15 @@ func NewXCTestConfiguration(
 		testIdentifiersToSkipEntry = createTestIdentifierSet(productModuleName, testsToSkip)
 	}
 
+	var appUnderTestExists = targetApplicationPath != "" && targetApplicationBundleID != ""
+	if appUnderTestExists {
+		contents["productModuleName"] = productModuleName
+		// contents["targetApplicationArguments"] = []interface{}{} //TODO: triggers a bug
+		contents["targetApplicationBundleID"] = targetApplicationBundleID
+		// contents["targetApplicationEnvironment"] = //TODO: triggers a bug map[string]interface{}{}
```

Comment:
Which bugs? That would be worth noting down 😉

Reply:
Those are really old lines from years ago. Removed to avoid confusion.